debug_line.c

```diff
@@ -25,7 +25,7 @@
 #include "debug_file.h"
 
 #if defined(MLX_WITH_LIBDW) /* MLX_WITH_LIBDW */
-#include <libdw.h>
+#include <elfutils/libdw.h>
```

Comment:
pkgconfig on linux doesn't resolve the include directory for libelf as /usr/include/elfutils, but instead as /usr/include. So to account for that, we need to modify the include statement.